src/js/modules/popupRegistration.js

```diff
@@ -25,20 +25,17 @@ function popupOpen() {
 
 function emptyPlaceCloseHandler(event) {
 	if (event.target === registerModalOverlay) {
-		popupClose(registerModalContent);
-		popupClose(registerModalGratitude);
+		popupsClose();
 	}
 }
 
 function closeBtnHandler() {
```

Comment:
why do you need `closeBtnHandler` if there is only `popupsClose` ?
use just `popupsClose` like a handler

Reply:
Done